src/services/swap_service.ts

```diff
@@ -121,6 +134,34 @@ export class SwapService {
         const gas = await estimateGasPromise;
         return new BigNumber(gas);
     }
+
+    // tslint:disable-next-line:prefer-function-over-method
+    private _attributeSwapQuoteOrders(
+        swapQuote: MarketSellSwapQuote | MarketBuySwapQuote,
+    ): MarketSellSwapQuote | MarketBuySwapQuote {
+        // Where possible, attribute any fills of these orders to the Fee Recipient Address
+        const contractAddresses = getContractAddressesForChainOrThrow(CHAIN_ID);
+        const bridgeMakerAddresses = [
+            contractAddresses.eth2DaiBridge,
+            contractAddresses.uniswapBridge,
+            contractAddresses.kyberBridge,
+        ];
```

Comment:
Is there not a proxyId for bridge asset datas?

Reply:
HECK YEAH AND NON ASYNC